src/appshell/qml/ProjectPage/ProjectPage.qml

```diff
@@ -163,23 +172,36 @@ DockPage {
 
             navigationSection: root.navigationPanelSec(tracksPanel.location)
 
-            width: root.verticalPanelDefaultWidth
-            minimumWidth: root.verticalPanelDefaultWidth
-            maximumWidth: root.verticalPanelDefaultWidth
+            width: root.verticalPanelDefaultWidth + (showEffectsSection ? effectsSectionWidth : 0)
+            minimumWidth: width
+            maximumWidth: width
 
             groupName: root.verticalPanelsGroup
 
             dropDestinations: root.verticalPanelDropDestinations
 
             titleBar: TracksTitleBar {
+                id: titleBarItem
+                effectsSectionWidth: tracksPanel.effectsSectionWidth
+                showEffectsSection: tracksPanel.showEffectsSection
+
                 onAddRequested: function(type) {
                     tracksPanel.add(type)
                 }
+                onEffectsSectionCloseButtonClicked: {
+                    tracksPanel.showEffectsSection = false
+                }
             }
 
             TracksPanel {
                 id: tp
                 navigationSection: tracksPanel.navigationSection
```

Comment:
please add new line before this line

Reply:
I wasn't the author, I'm certainly not going to begin applying rules I don't hold for important to code I haven't modified.